src/main/scala/diplomacy/LazyModule.scala

```diff
@@ -262,9 +407,13 @@ trait ModuleValue[T]
 
 object InModuleBody
```

Comment:
Shouldn't there be some Scaladoc for `InModuleBody` itself? or is that not normal scaladoc style, we just put it on the apply methods...?

Reply:
Generally it's better to put on the `object` in my opinion, especially for `objects` that we're effectively treating as functions in that they only have `apply` methods

Reply:
I fixed this.